### PR TITLE
Added Fabric and Kotlin Link

### PR DESCRIPTION
**Pull request by Apply55gx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ _Ads currently not available._
 - [**Toasty**](https://github.com/GrenderG/Toasty) for displaying error/success messages
 - [**ShapedImageView**](https://github.com/gavinliu/ShapedImageView) for round avatars
 - [**Material-About-Library**](https://github.com/daniel-stoneuk/material-about-library) for the about screen
-- **Fabric** analytics & crash reporting.
```
Comment (k0shk0sh) [inline README.md]: that will take you to IOS?

Comment (Apply55gx) [reply README.md]: My bad. Fixed it.

Comment (k0shk0sh) [reply README.md]: @Apply55gx np, I'll merge it once travis finished building.

Thanks

**Comment by Apply55gx** [comment]
@k0shk0sh Can you merge this?

**Pull request merged**